adafruit_esp32spi/adafruit_esp32spi.py

```diff
@@ -473,6 +507,25 @@ def connect_AP(self, ssid, password): # pylint: disable=invalid-name
             raise RuntimeError("No such ssid", ssid)
         raise RuntimeError("Unknown error 0x%02X" % stat)
 
+    def create_ap(self, ssid, password, channel=b'\x01'):
```

Comment:
Why do you need this method if you already have `wifi_set_ap_passphrase` and `wifi_set_ap_passphrase`? 

If those two methods are not meant to be user-accessable, and only called from within`create_ap`, consider changing them to private methods.

Reply:
Yeah, I can change them to be private methods, they're not meant to be user called. This method (`create_ap`) will call `wifi_set_ap_passphrase` or `wifi_set_ap_network` depending on if you pass in a password or not to `create_ap`. The benefit here is after calling it will wait until the we see that esp32 has switches its status to `AP_LISTENING` to verify the call worked.

Reply:
Ok, I see the utility of the two sep. methods.